File: Reverse.java
import java.util.Scanner;
public class Reverse{
    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter no of elements ");
        int n = sc.nextInt();
        System.out.println("Enter the elements");
        int[] arr = new int[n];
        System.out.println("original array");
        for(int i =0;i<arr.length;i++)
        {
            arr[i]=sc.nextInt();
            //System.out.println("original array");
            System.out.println( arr[i]+" ");
        }
        System.out.println("Reversed array");
        for(int i= arr.length-1;i>=0;i--)
        {
            
            System.out.println( arr[i]+" ");
        }
        

    }
}

File: array/reverse.cpp
 #include<iostream>
 using namespace std;
 int reverse(int arrA[],int n)
 {
       int arrB[50],i,j;
       for(i=n-1, j=0 ;i>=0; i--,j++)
       {
            // for(j=0; j>=n;j++)
             //{


             arrB[j]=arrA[i];
             }
       //}
       for(i=0;i<n;i++)
       {
             arrA[i]=arrB[i];
       }
 }

 int main()
 {
       int n,i,arrA[50];
       cout<<"enter no of elements";
       cin>>n;
       cout<<"enter elements";
       for( i=0;i<n;i++)
       {
             cin>>arrA[i];
       }
     reverse(arrA,n);
     cout<<"reverse array"<<endl;
     for(i=0;i<n;i++)
     {
         cout<< arrA[i]<<endl;
     }

       return 0;
 }
